src/helpers/url.tsx

```
@@ -140,3 +140,111 @@ export function removeQueryParamAndRedirect(
+			// Assuming Farcaster uses a pattern like 'farcaster.xyz/username'
+			return extractUsernameFromPattern(
+				cleanedUrl,
+				/farcaster\.xyz\/([^\/]+)/,
+			);
+		case 'lens':
+			// Assuming Lens uses a pattern like 'lens.xyz/username'
+			return extractUsernameFromPattern(
+				cleanedUrl,
+				/lens\.xyz\/([^\/]+)/,
+			);
+		case 'website':
```

Biome

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)